src/js/apps/patients/sidebar/action-sidebar_app.js

```diff
@@ -224,5 +232,26 @@ export default App.extend(extend({
   onRemoveAttachment(model) {
     model.destroy();
   },
+  addSubscriptions() {
+    const commentSubscriptions = [];
+
+    this.activityCollection.each(model => {
+      if (model.get('message')) {
```

Comment:
maybe `const commentSubscripts = this.activityCollection.filter({ type: 'comments' })`

Reply:
however we build this collection in the `onStart`.. we could probably just attach the comments separately rather than filter them each time.  We just need to add to both.